src/main/resources/db/migration/V20230314_21__IJPS-21_fee .sql

```diff
@@ -0,0 +1,10 @@
+create table public.fee (
+                             fee_record_id bigserial constraint pk_fee_record primary key,
+                             hmcts_service_id varchar(60) not null,
+                             judge_role_type_id varchar(60) not null,
+                             standard_fee numeric not null,
+                             higher_threshold_fee numeric,
+                             london_weighted_fee numeric,
```

Comment:
think we will need these as decimal now the java layer accepts BigDecimal

Reply:
As per the postgres documentation numeric and decimal are equivalent. - https://www.postgresql.org/docs/9.3/datatype-numeric.html